tensorboard/webapp/app_container.css

```diff
@@ -21,13 +21,16 @@ body {
 }
 
 :host {
+  background: #f5f5f5;
   display: flex;
   flex-direction: column;
   height: 100%;
 }
 
 app-header {
+  box-shadow: 0 1px 3px 3px rgba(0, 0, 0, 0.25);
   flex: 0 0;
+  z-index: 1;
```

Comment:
I'm curious: why add this?

Reply:
the shadow need to protrude the body but body is rendered below header (DOM structure wise).